src/tictactoe_server/app.clj

```diff
@@ -1,12 +1,13 @@
 (ns tictactoe-server.app
-  (:require [webserver.response :as response]
+  (:require [cob-app.get]
+            [cob-app.core :as backing-app]
```

Comment:
Maybe something in the webserver project isn't in the right spot, but shouldn't the `cob-app` just be some configuration to pass cob-spec and all the reusable stuff should be in the webserver right? It seems off to have your ttt server depend on your cob-app namespace